plugin/src/main/groovy/com/novoda/staticanalysis/internal/VariantAware.groovy

```diff
@@ -0,0 +1,37 @@
+package com.novoda.staticanalysis.internal
+
+import org.gradle.api.DomainObjectSet
+import org.gradle.api.NamedDomainObjectSet
+
+trait VariantAware {
+
+    Closure<Boolean> includeVariantsFilter
+
+    final variantSelector = { variants ->
```

Comment:
can this just be a normal private method then? it's weird to see it defined  it like this...

Reply:
Method didn't work how I wanted it to be with this trait thingy. I guess I can do Tobi's suggestion and make this a real collaborator class.